src/wallet/KintoWalletFactory.sol

```diff
@@ -266,3 +266,7 @@ contract KintoWalletFactory is Initializable, UUPSUpgradeable, OwnableUpgradeabl
         return created;
     }
 }
+
+contract KintoWalletFactoryV2 is KintoWalletFactory {
```

Comment:
I can see you've put the V2 here. But, in the case of the KintoWallet, you've put the V2 on the deployment script.

Not sure what's the best practice. I lean towards doing like this factory. 

Also, not sure if we would rather keep adding the same contract or creating a new file for each new upgrade.

We could have something like:
`/factories/KintoWalletFactory.sol`  
`/factories/KintoWalletFactoryV2.sol`  
`/wallets/KintoWallet.sol`  
`/wallets/KintoWalletV2.sol`  

thoughts?

Reply:
because V2 in the wallet is old. Wallet is already in V3. I like to keep in the file the current version. Its the system we had at Babylon